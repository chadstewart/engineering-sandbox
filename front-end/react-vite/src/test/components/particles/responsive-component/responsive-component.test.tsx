/**
 * @vitest-environment happy-dom
 */

import ResponsiveComponent from "@/components/particle/responsive-component/responsive-component";
import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";

describe("Particle Component: ResponsiveComponent", () => {
  it("Should match the snapshot", () => {
    const {
      container: { firstChild: variableToSnapshot }
    } = render(<ResponsiveComponent children={() => <></>} />);

    expect(variableToSnapshot).toMatchSnapshot();
  });
});
