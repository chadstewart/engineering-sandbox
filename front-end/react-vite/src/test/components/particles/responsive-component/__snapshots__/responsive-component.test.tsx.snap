// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Particle Component: ResponsiveComponent > Should match the snapshot 1`] = `null`;
